### ISSUE TITLE
No options

### ISSUE THREAD

**Issue by disha123** [Dgame/atom-php-fmt#13]
I have added php-amt package to my Atom 1.18.0 x64 with PHP 5.6.8 and i get only php-fmt:transform option; no other option is there. I also have downloaded fmt.phar and added to PHP. Please help.

OS: Windows 10
Atom: 1.18.0 x64
PHP: 5.6.8
fmt.phar: https://github.com/akalongman/fmt/blob/master/fmt.phar
fmt.phar location: D:\xampp\php\fmt.phar

**Comment by Dgame** [comment]
Sorry, I don't support this package anymore. I recommend VSCode and PHP 7.1.

**Comment by oxk4r01** [comment]
So, why don't you remove it from the Atom downloadable packages?

**Comment by Dgame** [comment]
I've thought I did that a long time ago. If not, I will catch that up later.

**Comment by oxk4r01** [comment]
It's a pity, because one of the great weak points of Atom, in my opinion, and the main reason don't use it as the main, is the lack of good code formatters, specially in the case of php, whose options are reduced to one or two and none of them works properly . Thanks for your past efforts anyway.

**Comment by Dgame** [comment]
My personal experience is, that Atom is awfully slow and consumes an comparable amount of memory compared to PHPStorm, but without the benefits of an real IDE. I'm using Sublime Text 3 and VSCode as Editors and can really recommend them.

**Comment by oxk4r01** [comment]
I agree. Slowness is other of the main Atom inconveniences, and agree to with the fact you're pointing of the lack of solid project management. PhpStorm is great but not free. Sublime, in my opinion, is not that rich in features, and, same as Atom, has a json config per file, very laborious and hard to maintain. VsCode dont know in deep. I actually use NetBeans, which I like a lot, and, although not perfect, is very suitable for php and the like.
